Source: Anders Olofsson, zimous, Peter Alfredsen, maybe others too.
Upstream: Probably fixed in next release.
Reason: gcc-4.3 and glibc-2.8 errors, bugs #218814 and #228373
diff -NrU5 licq-1.3.5.orig/plugins/auto-reply/configure.ac licq-1.3.5/plugins/auto-reply/configure.ac
--- licq-1.3.5.orig/plugins/auto-reply/configure.ac	2008-06-21 14:49:20.000000000 +0200
+++ licq-1.3.5/plugins/auto-reply/configure.ac	2008-06-21 14:52:18.000000000 +0200
@@ -44,11 +44,11 @@
 
 dnl Switch to C++ mode and check for needed C++ headers
 AC_LANG_SAVE
 AC_LANG_CPLUSPLUS
 
-AC_CHECK_HEADER(vector.h,,
+AC_CHECK_HEADER(vector,,
   AC_MSG_ERROR(You need to have the libstdc++ headers installed))
 
 AC_LANG_RESTORE
 
 AC_OUTPUT(
diff -NrU5 licq-1.3.5.orig/plugins/auto-reply/src/autoreply.cpp licq-1.3.5/plugins/auto-reply/src/autoreply.cpp
--- licq-1.3.5.orig/plugins/auto-reply/src/autoreply.cpp	2008-06-21 14:49:20.000000000 +0200
+++ licq-1.3.5/plugins/auto-reply/src/autoreply.cpp	2008-06-21 14:52:18.000000000 +0200
@@ -10,10 +10,11 @@
 #include <ctype.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <sys/wait.h>
 #include <signal.h>
+#include <climits>
 #ifdef HAVE_ERRNO_H
 #include <errno.h>
 #else
 extern int errno;
 #endif
diff -NrU5 licq-1.3.5.orig/plugins/email/configure.ac licq-1.3.5/plugins/email/configure.ac
--- licq-1.3.5.orig/plugins/email/configure.ac	2008-06-21 14:49:20.000000000 +0200
+++ licq-1.3.5/plugins/email/configure.ac	2008-06-21 14:52:18.000000000 +0200
@@ -46,11 +46,11 @@
 
 dnl Switch to C++ mode and check for needed C++ headers
 AC_LANG_SAVE
 AC_LANG_CPLUSPLUS
 
-AC_CHECK_HEADER(vector.h,,
+AC_CHECK_HEADER(vector,,
   AC_MSG_ERROR(You need to have the libstdc++ headers installed))
 
 AC_LANG_RESTORE
 
 AC_OUTPUT(
diff -NrU5 licq-1.3.5.orig/plugins/email/src/forwarder.cpp licq-1.3.5/plugins/email/src/forwarder.cpp
--- licq-1.3.5.orig/plugins/email/src/forwarder.cpp	2008-06-21 14:49:20.000000000 +0200
+++ licq-1.3.5/plugins/email/src/forwarder.cpp	2008-06-21 15:17:53.000000000 +0200
@@ -7,10 +7,11 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <string.h>
 #include <ctype.h>
 #include <stdio.h>
+#include <climits>
 #ifdef HAVE_ERRNO_H
 #include <errno.h>
 #else
 extern int errno;
 #endif
diff -NrU5 licq-1.3.5.orig/plugins/msn/configure.ac licq-1.3.5/plugins/msn/configure.ac
--- licq-1.3.5.orig/plugins/msn/configure.ac	2008-06-21 14:49:20.000000000 +0200
+++ licq-1.3.5/plugins/msn/configure.ac	2008-06-21 14:52:18.000000000 +0200
@@ -51,11 +51,11 @@
 
 dnl Switch to C++ mode and check for needed C++ headers
 AC_LANG_SAVE
 AC_LANG_CPLUSPLUS
 
-AC_CHECK_HEADER(vector.h,,
+AC_CHECK_HEADER(vector,,
   AC_MSG_ERROR(You need to have the libstdc++ headers installed))
 
 AC_LANG_RESTORE
 
 msn_gcc_major_version=0
diff -NrU5 licq-1.3.5.orig/plugins/rms/src/rms.cpp licq-1.3.5/plugins/rms/src/rms.cpp
--- licq-1.3.5.orig/plugins/rms/src/rms.cpp	2008-06-21 14:49:20.000000000 +0200
+++ licq-1.3.5/plugins/rms/src/rms.cpp	2008-06-21 15:08:27.000000000 +0200
@@ -7,10 +7,11 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <string.h>
 #include <ctype.h>
 #include <stdio.h>
+#include <climits>
 #ifdef HAVE_ERRNO_H
 #include <errno.h>
 #else
 extern int errno;
 #endif
diff -NrU5 licq-1.3.5.orig/src/fifo.cpp licq-1.3.5/src/fifo.cpp
--- licq-1.3.5.orig/src/fifo.cpp	2008-06-21 14:49:20.000000000 +0200
+++ licq-1.3.5/src/fifo.cpp	2008-06-21 14:52:18.000000000 +0200
@@ -28,10 +28,11 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <sys/stat.h>
 #include <fcntl.h>
 #include <ctype.h>
+#include <climits>
 #include "time-fix.h"
 
 #ifdef HAVE_ERRNO_H
 #include <errno.h>
 #else
